lib/trino-metastore/src/main/java/io/trino/metastore/StorageFormat.java

```diff
@@ -28,18 +28,20 @@
 @Immutable
 public class StorageFormat
 {
-    public static final StorageFormat NULL_STORAGE_FORMAT = new StorageFormat(null, null, null);
+    public static final StorageFormat NULL_STORAGE_FORMAT = new StorageFormat(null, null, null, null);
     public static final StorageFormat VIEW_STORAGE_FORMAT = NULL_STORAGE_FORMAT;
 
     private final String serde;
     private final String inputFormat;
     private final String outputFormat;
+    private final String fileExtension;
 
-    private StorageFormat(String serde, String inputFormat, String outputFormat)
+    private StorageFormat(String serde, String inputFormat, String outputFormat, String fileExtension)
     {
         this.serde = serde;
         this.inputFormat = inputFormat;
         this.outputFormat = outputFormat;
+        this.fileExtension = fileExtension;
```

Comment:
rnn here (and for other fields, in a separate commit)

Reply:
I believe all parameters are nullable.